hail/src/main/scala/is/hail/variant/Call.scala

```diff
@@ -95,13 +95,18 @@ object Call extends Serializable {
 
   def alleleRepr(c: Call): Int = c >>> 3
 
-  def allelePair(c: Call): AllelePair = {
+  def allelePair(c: Call): Int = {
```

Comment:
Changes here and in Genotype.scala are to remove the AllelePair value class, which made code generation much harder and also introduced unintentional allocation in at least once place.